client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
```

Comment:
In the future I would just add all of this to the main gitignore, just so you only have one gitignore file.